bodhi/server/templates/override.html

```diff
@@ -20,14 +21,26 @@
   </div>
 
   <script>$("#js-warning").addClass('hidden');</script>
-
-  <div id="new-override-form" class="col-md-12 hidden m-t-3">
+%endif
+  %if override is UNDEFINED:
+  <h2 class="pull-left m-t-3">New Override</h2>
```

Comment:
this page is also shown for creating a new override. The override is undefined in this case, and this is how we check in the template if we are looking at the new override form or not.